src/main/java/uk/gov/digital/ho/hocs/casework/api/StageService.java

```diff
@@ -355,15 +355,22 @@ private Stage getStage(UUID caseUUID, UUID stageUUID) {
         }
     }
 
+    private BasicStage getBasicStage(UUID caseUUID, UUID stageUUID) {
```

Comment:
Name needs to reflect its purpose. Perhaps renaming the other version is more appropriate i.e. `getStageTypeAndCaseFromStageData()` ?

Reply:
Good suggestion